onnxruntime/contrib_ops/cuda/bert/attention_data.h

```diff
@@ -180,6 +180,35 @@ struct GroupQueryAttentionData {
   bool use_memory_efficient_attention = false;
 };
 
+template <typename T>
+struct PagedAttentionData {
+  // Input Tensors
+  const T* query = nullptr;
+  const T* key = nullptr;
+  const T* value = nullptr;
+  T* key_cache = nullptr;
+  T* value_cache = nullptr;
+  const int* cumulative_sequence_length = nullptr;
```

Comment:
what is different of cumulative_sequence_length and cumulative_seqlens_kv? It is better to distinguish them from naming like cumulative_seqlens_q and cumulative_seqlens_kv.